src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/prisonperson/ReconciliationService.kt

```diff
@@ -5,21 +5,24 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.async
 import kotlinx.coroutines.awaitAll
 import kotlinx.coroutines.withContext
+import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.config.trackEvent
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.nomissync.model.PrisonPersonReconciliationResponse
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.prisonperson.model.PhysicalAttributesDto
-import uk.gov.justice.digital.hmpps.prisonertonomisupdate.services.NomisApiService
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.prisonperson.physicalattributes.DpsApiService
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.services.asPages
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.services.awaitBoth
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.services.doApiCallWithRetries
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.prisonperson.NomisApiService as PrisonPersonNomisApiService
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.services.NomisApiService as PrisonerNomisApiService
 
-@Service
-class PrisonPersonReconService(
-  private val dpsApi: PrisonPersonDpsApiService,
-  private val prisonPersonNomisApi: PrisonPersonNomisApiService,
-  private val nomisApi: NomisApiService,
+@Service("prisonPersonReconciliationService")
+class ReconciliationService(
+  @Qualifier("physicalAttributesDpsApiService") private val dpsApi: DpsApiService,
```

Comment:
This is an anomaly - we have to call the DPS physical attributes endpoint when doing the reconciliation. With this approach we'll end up calling a lot of services for all of the different fields.

I'll suggest that we have a dedicated reconciliation endpoint, the same as we do for getting the NOMIS reconciliation data.